falcon/response.py

```diff
@@ -123,6 +133,103 @@ def set_stream(self, stream, stream_len):
         self.stream = stream
         self.stream_len = stream_len
 
+    def set_cookie(self, key, value, **cookie_params):
+        """Set a response cookie.
+
+        Args:
+            key (str): Cookie key name
```

Comment:
Including "key" in the description seems like it is exposing an implementation detail. What do you think about just saying "Cookie name"?